### ISSUE TITLE
DIGITALTWIN - Feedback Round 1 (deadline 7 April 2021)

### ISSUE THREAD

**Issue by AlexandreHannelas** [threefoldfoundation/www_threefold_twin#61]
- Create a feedback form 
- Send DT trial version link to people inside and outside Threefold 
- Gather feedback
- Report on what needs to be improved, what is not working, and what is good. 
- Coordinate with Product Management and Jimber Team 
- Create issues on Github

**Comment by nbabenko** [comment]
first round of feedback is done. Message distributed:

Hi everyone!

🚀🚀 Our DigitalTwin app is in beta mode right now and we would really appreciate your feedback and insights. 🚀🚀

🍭 Please go and try it: https://digitaltwin.jimbertesting.be/ 
🤓 Here are the manual how to get started: https://new.threefold.io/info/twin#/twin__getting_started

🤸🏻‍♀️ Join our DigitalTwin Early Adopters chat if you want to hang out or have questions: https://t.me/joinchat/EgbQdD5DJzk4YmFj
🎤 Or ask your questions on our forum: https://forum.threefold.io/t/beta-mode-available-feedback-are-welcome/738

📋 Please share your feedback with us here: https://docs.google.com/forms/d/e/1FAIpQLSfkmFjjiN5wRm8c5DVR5jzLUebSqFPPPPSpFFj1i0vlGTL6EA/viewform

**Comment by nbabenko** [comment]
most of the reported issues were already Known, Ken registered everything. Waiting for the next releases before we can continue with collecting feedback.

**Comment by nbabenko** [comment]
initial feedback gathered. Creating new task for the next round.

**Comment by nbabenko** [comment]
https://github.com/threefoldfoundation/home/issues/166
